alvr/client_openxr/src/interaction.rs

```diff
@@ -242,84 +272,90 @@ pub fn initialize_interaction(
         .as_ref()
         .map(|s| s.eye_tracking_fb)
         .unwrap_or(false)
-        && xr_ctx.instance.exts().fb_eye_tracking_social.is_some()
         && xr_ctx
-            .instance
-            .supports_social_eye_tracking(xr_ctx.system)
-            .unwrap())
-    .then(|| xr_ctx.session.create_eye_tracker_social().unwrap());
+            .extra_extensions
+            .supports_social_eye_tracking(&xr_ctx.instance, xr_ctx.system))
+    .then(|| {
+        xr_ctx
+            .extra_extensions
+            .create_eye_tracker_social(xr_ctx.session.clone())
+            .unwrap()
+    });
 
     let face_tracker_fb = (face_tracking_sources
         .as_ref()
         .map(|s| s.face_tracking_fb)
         .unwrap_or(false)
-        && xr_ctx.instance.exts().fb_face_tracking2.is_some()
         && xr_ctx
-            .instance
-            .supports_fb_visual_face_tracking(xr_ctx.system)
-            .unwrap()
+            .extra_extensions
+            .supports_fb_visual_face_tracking(&xr_ctx.instance, xr_ctx.system)
         && xr_ctx
-            .instance
-            .supports_fb_audio_face_tracking(xr_ctx.system)
-            .unwrap())
-    .then(|| xr_ctx.session.create_face_tracker2_fb(true, true).unwrap());
+            .extra_extensions
+            .supports_fb_audio_face_tracking(&xr_ctx.instance, xr_ctx.system))
+    .then(|| {
+        xr_ctx
+            .extra_extensions
+            .create_face_tracker2_fb(&xr_ctx.session, true, true)
```

Comment:
This should just use whatever is available

Reply:
I would say no, because I think we only want face tracking when we have visual support (quest pro only). Quest Pro of course supports also audio. But for the other headsets there is no real benefit of us providing face tracking data from only audio while games already support visemes.